client/src/app/tabs/cloud-bpmn/BpmnEditor.js

```diff
@@ -849,6 +869,11 @@ export class BpmnEditor extends CachedComponent {
       elementTemplateChooser: false,
       keyboard: {
         bind: false
+      },
+      elementTemplates: {
+        engines: {
+          camundaModeler: version
```

Comment:
Let's change this engine to `camundaDesktopModeler`. I assume that the version is the current Desktop Modeler version, i.e. `v5.30.0`.